src/main/java/com/tstepnik/planner/domain/task/Task.java

```diff
@@ -12,7 +12,7 @@
     private Long id;
 
     @NotEmpty
-    @Size(min=3)
+    @Size(min=3,max = 255)
```

Comment:
makes sense :)

Reply:
You're looking at wrong commit, check the lastone before your request changes.

Reply:
What are you talking about?

Reply:
I changed it before request changes from you.